main.go

```diff
@@ -120,11 +121,15 @@ type AppState struct {
 	Settings
 	ArborState
 	*sprigTheme.Theme
-	DataDir string
+	DataDir      string
+	TimeLaunched uint64
 }
 
 func NewAppState(dataDir string) (*AppState, error) {
 	dataDir = filepath.Join(dataDir, "sprig")
+	// NOTE: time has to be converted from nanoseconds to milliseconds for Arbor's specifications
```

Comment:
This comment can be removed if desired. I added it to explain the random math in line 132.

Reply:
I appreciate the diligence! The comment can stay.